scripts/dev/install-pnpm.sh

```diff
@@ -1,25 +1,6 @@
 #!/usr/bin/env bash
 
-if [[ $OSTYPE == 'darwin'* ]]; then
-  echo 'This script is intended to be run on Linux only'
-  exit 1
-fi
+# We can't install pnpm using node because pnpm 8 isn't compatible with node 14, which is what
+# we use on lambda. Instead, we can install a standalone binary that doesn't depend on node.
 
-# The pnpm cdn seems to be quite flakey, so let's retry a few times before
-# erroring out.
-
-MAX_RETRIES=5
-RETRY_DELAY_SECS=15
-RETRY=0
-
-while true; do
-  curl -sSfL https://pnpm.js.org/pnpm.js | sudo node - add --global pnpm@7 && break
-  ERROR=$?
-  RETRY=$((RETRY + 1))
-  if [ "${RETRY}" -gt ${MAX_RETRIES} ]; then
-    log error "Bailing out after ${RETRY} retries... ¯\_(ツ)_/¯"
-    exit ${ERROR}
-  fi
-  log warn "Waiting to retry ${RETRY} of ${MAX_RETRIES}..."
-  sleep ${RETRY_DELAY_SECS}
-done
+sudo npm install -g @pnpm/exe@~8.5.1
```

Comment:
this is a bit aggressive but I guess we have no option?

Reply:
Is it an option to stay with a legacy version of pnpm and renovate?

Reply:
The `sudo` is an attempt to fix a CI error. It wasn't necessary for running locally on my mac. If it works with `sudo` on CI I could condition that to only happen on CI

Reply:
Oh I meant the whole `npm install -g` thing.

Reply:
Ah, well I think there's a bit of a chicken-egg problem there where we need `pnpm` to install node deps but we can't install it locally without `pnpm` because then the `node_modules` layout is already wrong an `pnpm` doesn't know what to do with it. Installing pnpm globally instead of locally is one way around that